### ISSUE TITLE
Enclosure does not fit M2 received from Kickstarter campaign.

### ISSUE THREAD

**Issue by Jfunct** [macchina/m2-enclosures#3]
I printed the enclosure files in this repo, and there's a considerable amount of error in the screw boss positions relative to the mounting holes on the PCB.  I can't even get the enclosure to fit around the stack, because the bosses interfere with the corner cut-outs on the mezzanine board.

Has the PCB design changed mechanically since this enclosure CAD work was done?

**Comment by rocketjosh** [comment]
The PCB dimensions and hole locations did not change between R2 (beta) and R3 (release). However, we used a V-scored PCB in R3, so that may require a little more interior clearance in the enclosures. We've added a couple new files to this repo: 

M2R3_CASE_TOP_UTD (STL and Solidworks)
M2R3_CASE_BOTTOM_UTD (STL and Solidworks)

These parts should have a bit more slop on the inside walls to allow for that V-scored PCB edge. 

Give those a shot and let us know how they work out.
